webhook_server/tests/test_pull_request_handler.py

```
@@ -18,11 +20,21 @@
+    PRE_COMMIT_STR,
     TOX_STR,
     VERIFIED_LABEL_STR,
     WIP_STR,
 )
 
+T = TypeVar("T")
+
+
+# Async shim for mocking asyncio.to_thread in tests
+# This allows us to run sync functions in tests while preserving async/await semantics
+async def _sync_to_thread[T](func: Callable[..., T], *args: Any, **kwargs: Any) -> T:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)